lib/datasampler.rb

```diff
@@ -0,0 +1,215 @@
+# frozen_string_literal: true
+require_relative '../environment.rb'
+require_relative '../models.rb'
+require_relative './util.rb'
+
+###
+# DataImporter - methods for sampling production data for use in testing
+###
+class DataImporter
+  def initialize
+    @directory = 'test_data/'
+  end
+
+  # Deletes all the extant data. This does not drop tables, only removes rows
+  # rubocop:disable AbcSize
+  def delete_data
+    # deletes everything
+    Report.plugin_matrix.each do |resource_name, measurements|
+      model = Object.const_get(resource_name + 'Resource').camelcase(:upper)
```

Comment:
This causes:
NameError: wrong constant name dbResource
/data/code/lib/datasampler.rb:19:in `const_get'
/data/code/lib/datasampler.rb:19:in `block in delete_data'
/data/code/lib/datasampler.rb:18:in `each'
/data/code/lib/datasampler.rb:18:in `delete_data'
/data/code/lib/datasampler.rb:79:in `import_data'
/data/code/Rakefile:82:in `block in <top (required)>'
Tasks: TOP => import_data 

when running `rake import_data`